pydoctor/templatewriter/pages/attributechild.py

```diff
@@ -71,7 +71,7 @@ def functionBody(self, request: object, tag: Tag) -> "Flattenable":
 
     @renderer
     def constantValue(self, request: object, tag: Tag) -> "Flattenable":
-        if self.ob.kind is not DocumentableKind.CONSTANT or self.ob.value is None:
+        if self.ob.kind not in (DocumentableKind.CONSTANT, DocumentableKind.TYPING_VAR) or self.ob.value is None:
```

Comment:
Maybe we should have a system method determining which object we should show the value for, this would have the benefit to be hackable.

```suggestion
        if not self.ob.sytem._should_present_constant_value_of(self.ob) or self.ob.value is None:
```